src/Administration/Resources/app/administration/src/app/mixin/listing.mixin.ts

```diff
@@ -38,6 +38,7 @@ export default Shopware.Mixin.register(
             entitySearchable: boolean;
             freshSearchTerm: boolean;
             previousRouteName: string;
+            _cachedSearchRankingFields: any;
```

Comment:
this could be specifc Object<key: string, value: string> or something like that, the getSearchFieldsByEntity will return something like { "name" 5000, "description": 3000, etc }